### PR TITLE
[Student][Automation][MBL-13150]: Fixed issues in displaysBookmarkMenuItem tests

### PR DESCRIPTION
**Pull request by joehoag**
The displaysBookmarkMenuItem test in AssignmentDetailsRenderTest was failing regularly on APIs 27 and 28.  It looks like the root cause was that, for those APIs, the content description for the overflow menu can be some control characters in addition to "More options", and so we had to test for "contains 'More options'" rather than "is 'More options'" in a few places to get our logic working.

(But it only showed this weird behavior on FTL devices!)

Along the way, I added some logic to enable a11y checking for ALL tests in AssignmentDetailsRenderTest, and fixed up a couple of a11y issues in the process.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/student/src/androidTest/java/com/instructure/student/ui/utils/StudentTest.kt [hunk 1]
```diff
@@ -53,4 +57,10 @@ abstract class StudentTest : CanvasTest() {
     val peopleListPage = PeopleListPage()
     val personDetailsPage = PersonDetailsPage()
     val modulesPage = ModulesPage()
+    val syllabusPage = SyllabusPage()
+
+    // A no-op interaction to afford us an easy, harmless way to get a11y checking to trigger.
+    fun meaninglessSwipe() {
```
Comment (joehoag) [inline apps/student/src/androidTest/java/com/instructure/student/ui/utils/StudentTest.kt]: I'm open to alternative names for this function.  But this function should make it easy for any test writer to trigger a11y checks without exercising any serious functionality.

File: apps/student/src/main/res/layout/fragment_assignment_details.xml [hunk 2]
```diff
@@ -627,7 +627,7 @@
                             android:layout_height="wrap_content"
                             android:layout_marginStart="8dp"
                             android:layout_marginEnd="8dp"
-                            android:minHeight="24dp"
+                            android:minHeight="48dp"
                             android:scrollbars="none"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/fragment_assignment_details.xml]: Accessibility fixes.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005239&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt [hunk 3]
```diff
@@ -135,9 +140,38 @@ abstract class CanvasTest : InstructureTest(BuildConfig.GLOBAL_DITTO_MODE) {
             override fun matches(item: Any): Boolean {
                 when(item) {
                     is AccessibilityViewCheckResult -> {
-                        return item.view.width < dim && item.view.height >= dim
+                        val result = item.view.width < dim && item.view.height >= dim
+                        //Log.v("overflowWidth", "view=${getResourceName(item.view)}, desc=${item.view.contentDescription}, w=${item.view.width}, h=${item.view.height}, res=$result")
```
Comment (Blackchai) [inline automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt]: Do we want all of these logs to be commented out?

Comment (joehoag) [reply automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt]: They were (are) very useful for debugging.  But as they are very verbose and provide little utility once things are working, I think that it's best to keep them commented out.

If we run into further problems related to the logged logic, I'll try to find a way to log the info in a not-so-verbose manner.

**Pull request merged**